Update artifacts.md (#3560)
dbt-labs/docs.getdbt.com: website/docs/docs/deploy/artifacts.md

### Documentation

> When set up, dbt Cloud updates the **Documentation** link in the header tab so it links to documentation for this job. This link always points to the latest version of the documentation for your project! Please note both the job's commands and docs generate step (triggered via the checkbox) must be successful within the job invocation to populate or update project-level documentation. 

Reviewer: ```suggestion
When set up, dbt Cloud updates the **Documentation** link in the header tab so it links to documentation for this job. This link always directs you to the latest version of the documentation for your project. 

Note that both the job's commands and the docs generate step (triggered by the **Generate docs on run** checkbox) must succeed during the job invocation for the project-level documentation to be populated or updated.

```
